src/Core/src/Platform/Windows/WindowRootView.cs

```diff
@@ -231,6 +247,90 @@ void LoadAppTitleBarControls()
 			UpdateAppTitleBarMargins();
 		}
 
+		/// <summary>
+		/// This is the default app title bar, which overrides `MauiAppTitleBarTemplateDefault` in `WindowRootViewStyle.xaml`
+		/// </summary>
+		/// <returns></returns>
+		private Grid CreateAppTitleBar()
```

Comment:
It looks like @mattleibow is going to review a fix for SR2/SR3 timeline so that we can modify our XAML files. Let's see how that goes before getting to creative here.